src/Compilers/Server/VBCSCompiler/ServerDispatcher.cs

```diff
@@ -102,20 +102,29 @@ public void ListenAndDispatchConnections(TimeSpan? keepAlive, CancellationToken
                     _clientConnectionHost.EndListening();
                 }
 
-                // This type is responsible for cleaning up resources associated with _listenTask. Once EndListening
-                // is complete this task is guaranteed to be completed. If it ran to completion we need to 
-                // dispose of the value.
-                Console.WriteLine(_listenTask?.Status);
-                Debug.Assert(_listenTask is null || _listenTask.IsCompleted);
-                if (_listenTask?.Status == TaskStatus.RanToCompletion)
+                if (_listenTask is not null)
                 {
-                    try
+                    // This type is responsible for cleaning up resources associated with _listenTask. Once EndListening
+                    // is complete this task is guaranteed to be either completed or have a task scheduled to complete
+                    // it. If it ran to completion we need to dispose of the value.
+                    Console.WriteLine(_listenTask.Status);
```

Comment:
Please kill that `Console.WriteLine` for me 😄

Reply:
➡️ Done